packages/atomic/src/components/common/result-templates/result-template-common.tsx

```diff
@@ -164,3 +164,19 @@ export function makeMatchConditions(
   }
   return conditions;
 }
+
+export function makeDefinedConditions(
```

Comment:
As Luc pointed out, I think this function here could be easily unit tested

Reply:
Yes for straightforward `expected in -> expected out` kinda function we have UT in the project.

Does not need to be in the utils folder.

Reply:
I added some tests 👍